SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/OwnershipLockingPolicy/SpatialPackageMapClientMock.h

```diff
@@ -1,21 +0,0 @@
-// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
-
-#pragma once
-
-#include "EngineClasses/AbstractSpatialPackageMapClient.h"
-#include "WorkerSDK/improbable/c_worker.h"
-
-#include "SpatialPackageMapClientMock.generated.h"
-
-UCLASS()
-class USpatialPackageMapClientMock : public UAbstractSpatialPackageMapClient
```

Comment:
Do we need the AbstractSpatialPackageMapClient now? If not, could we remove it and the virtual definition of GetEntityIdFromObject().

Reply:
Can definitely remove yea, you good with @jennifer-at-improbable-io 's suggestion to make a ticket tracking the creation of StaticComponentView / PackageMap / Translator mocks which just links back here?

Reply:
Moving to a single location looks good, but if we don't need the virtual functionality, lets remove it until we do.